ui/admin/app/components/model-providers/ModelProviderBanner.tsx

```diff
@@ -0,0 +1,18 @@
+import { CircleAlert } from "lucide-react";
+
+import { TypographyP } from "~/components/Typography";
+
+export function ModelProviderBanner() {
```

Comment:
now that I look at it, could this entire component be replaced with the ShadCn `Alert` component?

https://ui.shadcn.com/docs/components/alert